Update Live recording docu (#76)
ing-bank/ingenious-doc: docs/gettingstarted.md

-----------------------

## **Quick Start with Recording** - <span style="color:#FF6200">**Live Playwright Recorder (CodeGen)**</span>  

As you interact with the browser, each action is captured and added to the Test Case editor in real time.

### Steps for recording

 * Launch **INGenious Playwright Studio**

 * Click on the **Record** icon in the Test Case toolbar

   ![record](img/recording/1.JPG "record")

 * The **Choose Recording Target** dialog appears. Pick where the recording should be saved:

    * **New test case under Test Scenario** — provide a **Scenario** name (defaults to `LiveRecordingScenario`) and a **Test case** name (defaults to `LiveRecordingTestCase`)
    * **New test case under Reusable Scenario** — provide a **Reusable scenario** name (defaults to `LiveRecordingReusable`) and a **Test case** name (defaults to `LiveRecordingReusableTestCase`)

>     * Click **Start Recording** to continue, or **Cancel** to abort.

Reviewer: ```suggestion
    * Click **Start Recording** to begin recording, or **Cancel** to close the dialog.
```

    ![ChooseRecordingDialogueBox](img/recording/ChooseRecordingDialogueBox.png "Choose Recording Target")

 * A loader will show up while the Playwright recorder is being loaded

!!! warning 
    

>     For **first time usage**, Playwright will download its required binaries. If network connection is slow, this may cause the recorder to time out.

Reviewer: ```suggestion
    **On first use**, Playwright downloads the required browser binaries. If the network connection is slow, the recorder may time out during setup.
```

>     Several dialog boxes might appear during setup, **just click 'OK' to proceed**. Once the recorder opens, **close it and start a new recording**.

Reviewer: ```suggestion
    Several dialog boxes may appear during setup. Click **OK** to continue. Once the recorder opens successfully, **close it and start a new recording session**.
```

 * INGenious creates/opens the target scenario and test case, then launches the **Playwright Inspector** together with a **Chromium** browser window.

>  * Once the recorder is ready, INGenious automatically hides its own recording console and minimizes the Playwright Inspector window, leaving the Chromium browser in focus so you can go straight into interacting with playwright recorder.

Reviewer: ```suggestion
 * Once the recorder is ready, INGenious automatically hides the recording console and minimizes the Playwright Inspector window. This leaves the Chromium browser in focus so you can immediately begin interacting with the Playwright recorder.
```

 * Every action you perform is streamed live into the **Test Case editor** as you go:

    * Each recorded step is inserted into the step grid immediately and highlighted in **green**, so you can see exactly what was just captured
    * Any web object you interact with is automatically added to the **Object Repository**, under a page named after your test case
    * A running log of captured steps (e.g. `Step 3 captured: Click on 'Login' [Login_button]`) is shown in the recording console

>  * When you're done, click the same toolbar icon again — it now shows **Stop Recording** — to end the session. This closes the Chromium browser and the Playwright process, and saves the recorded steps into your test case.

Reviewer: ```suggestion
 * When recording is complete, click the same toolbar icon again. The icon now displays **Stop Recording**. Clicking it ends the recording session. This closes the Chromium browser, stops the Playwright recording process, and saves the recorded steps to the test case.
```

    ![LiveRecording](img/recording/LiveRecording.gif "LiveRecording")

!!! tip

>     You can also close the Chromium browser window directly instead of clicking **Stop Recording** — INGenious detects this and finalizes the recording the same way. Using **Stop Recording** is recommended.

Reviewer: ```suggestion
    You can also close the Chromium browser window directly instead of clicking **Stop Recording**. INGenious detects this and finalizes the recording automatically.. Using **Stop Recording** is recommended.
```

>  * Your recording is now available as a fully populated **Scenario** (or **Reusable Scenario**) and **Test Case**, with all relevant test steps, web objects, and test data already in place, ready to run or edit further.

Reviewer: ```suggestion
 * Your recording is now available as a fully populated **Scenario** (or **Reusable Scenario**) and **Test Case**. All relevant test steps, web objects, and test data are automatically created and ready for execution or further editing.
```
